openmetadata-ui/src/main/resources/ui/cypress/common/DomainUtils.ts

```diff
@@ -49,7 +49,7 @@ export const validateDomainForm = () => {
     .type(INVALID_NAMES.MAX_LENGTH);
   cy.get('#name_help')
     .should('be.visible')
-    .contains(NAME_MAX_LENGTH_VALIDATION_ERROR);
+    .contains(NAME_MIN_MAX_LENGTH_VALIDATION_ERROR_1_128);
```

Comment:
this will not assert the value, this will only get the value, for assertion please use
`cy.get(<SELECTOR>).should("contain", <VALUE>)`